dkms.in

```diff
@@ -916,11 +916,19 @@ prepare_signing()
     fi
     echo "Public certificate (MOK): $mok_certificate"
```

Comment:
Nit: let's change the bug number from the commit summary to the message. Github supports "Fixes: #256" which will auto-close the issue.